tests/prague/eip7692_eof_v1/eip7620_eof_create/test_returncontract.py

```diff
@@ -0,0 +1,124 @@
+"""
+Tests for RETURNCONTRACT instruction validation
+"""
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.exceptions import EOFException
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-7620.md"
+REFERENCE_SPEC_VERSION = "f20b164b00ae5553f7536a6d7a83a0f254455e09"
+
+
+def test_returncontract_valid(
+    eof_test: EOFTestFiller,
+):
+    """Deploy container index 0"""
+    eof_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    code=Op.RETURNCONTRACT[0](0, 0),
+                ),
+                Section.Container(container=Container(sections=[Section.Code(code=Op.INVALID)])),
+            ],
+        )
+    )
+    """Deploy container index 1"""
+    eof_test(
```

Comment:
It looks like putting several `eof_test()` calls into one `test_*` function produces just one fixture with the last case.

Reply:
changed to a function per `eof_test()` call